src/lib.rs

```diff
@@ -167,19 +175,35 @@ impl Config {
         println!("cargo:rustc-link-search=native={}", dst.display());
         println!("cargo:rustc-link-lib=static={}",
                  &output[3..output.len() - 2]);
+
+        // Add specific C++ libraries, if enabled.
+        if self.cpp {
+            println!("cargo:rustc-link-lib=static=stdc++");
```

Comment:
Yeah I'd definitely believe that, but I think it may be best done as a dynamic dependency (e.g. removing `static=`) instead of a static one. Statically linking libstdc++ is pretty rare I believe.